crates/aiken-lang/src/tipo/infer.rs

```diff
@@ -348,7 +348,7 @@ fn infer_definition(
                                             annotation,
                                             location,
                                             tipo: t.clone(),
-                                            doc: None,
+                                            doc,
```

Comment:
This one is solely responsible for 3h of debug. 😢

Reply:
Ooff subtle but deadly